wgpu-core/src/device/mod.rs

```diff
@@ -53,7 +54,7 @@ fn own_label(label: &Label) -> String {
 }
 
 pub const MAX_COLOR_TARGETS: usize = 4;
-pub const MAX_MIP_LEVELS: usize = 16;
+pub const MAX_MIP_LEVELS: u32 = 16;
```

Comment:
Converted this to a `u32`, to match the type in the descriptor. It makes more sense to cast this to an `usize` in the texture state storage in `track/texture.rs`